src/action-menu/_ActionMenuHeader.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import ThemeContext from 'terra-theme-context';
+import Button from 'terra-button';
+import IconClose from 'terra-icon/lib/icon/IconClose';
+import { injectIntl, intlShape } from 'react-intl';
+
+import styles from './ActionMenu.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * The label text to display for the header.
+   */
+  label: PropTypes.string.isRequired,
+  /**
+   * Callback function for close button selection.
+   * Returns the event e.e. onClose(event).
+   */
+  onClose: PropTypes.func,
+  /**
+   * @private
+   * The intl object to be injected for translations.
+   */
+  intl: intlShape,
```

Comment:
Just a note: react-intl 5 doesn't offer up the shape and you will have to do something like this: https://github.com/cerner/terra-framework/blob/main/packages/terra-application-layout/src/header/_ApplicationHeader.jsx#L56

Doesn't really matter for this PR but will matter in terra-application v2